components/molecules/sidebar/__mocks__/links.js

```diff
@@ -1,32 +1,33 @@
 /**
  * Sidebar
  */
-export const Sidebar = [
-  {
-    name: 'Dashboard',
-    to: '/dashboard'
-  },
-  {
-    divider: true
-  },
-  {
-    name: 'Profile',
-    to: '/dashboard/profile'
-  },
-  {
-    name: 'Calendar',
-    to: '/dashboard/calendar'
-  },
-  {
-    divider: true
-  },
-  {
-    name: 'Messages',
-    to: '/dashboard/messages'
-  },
-  {
-    icon: ['far', 'bell'],
-    name: 'Notifications',
-    to: '/dashboard/notifications'
-  }
-]
+export const Sidebar = {
+  title: 'sideBar title',
+  items: [
+    {
+      name: 'Dashboard',
+      to: '/dashboard',
+      icon: ['fas', 'cloud']
```

Comment:
@esuarezz Alpha here
Icon is good and think with sub-components passing in a named object is a good patter I started on a few others

For instance the to is for Link right so with pattern we would do this.

link: {
  to:
}